expat/lib/xmltok.c

```diff
@@ -303,6 +304,9 @@ sb_charMatches(const ENCODING *enc, const char *p, int c) {
 #  define CHAR_MATCHES(enc, p, c) (*(p) == c)
 #endif
 
+/* Fallthrough statements are unreachable. */
+#define MAYBE_FALLTHROUGH
+
```

Comment:
I get the idea. My concern is the `MAYBE_` part of the name only, as it doesn't explain itself as much as I'd consider ideal. How about `FALLTHROUGH_IF_REACHABLE`, what do you think?

Reply:
Done.